release-controller/release_index.py

```diff
@@ -26,12 +24,12 @@ class Release(BaseModel):
     versions: List[Version]
 
 
-class ReleaseIndex(BaseModel):
+class Welcome4(BaseModel):
```

Comment:
what happened here?

Reply:
Apparently we have a `/bin/poetry.sh` which has `datamodel-codegen` which uses `release-index-schema.json` and generates that file. I will make changes in a separate PR to change the root node name to fix this